frontend/src/context/OrganizationContext/OrganizationContext.tsx

```diff
@@ -18,16 +19,21 @@ type Props = {
 };
 
 export const OrgProvider = ({ children }: Props): JSX.Element => {
+  const router = useRouter();
   const { currentWorkspace } = useWorkspace();
   const { data: userOrgs, isLoading } = useGetOrganization();
 
+  if (userOrgs?.length === 0){
+    router.push('/noOrganizations');
+  }
+
   const currentWsOrgID = currentWorkspace?.organization;
 
   // memorize the workspace details for the context
   const value = useMemo<TOrgContext>(
     () => ({
       orgs: userOrgs,
-      currentOrg: (userOrgs || []).find(({ _id }) => _id === currentWsOrgID) || (userOrgs || [])[0],
+      currentOrg: (userOrgs || []).find(({ _id }) => _id === localStorage.getItem('orgData.id')) || (userOrgs || [])[0],
```

Comment:
@akhilmhdh thoughts here?

I know you wanted to move away from using local storage in the future — Perhaps there's a better way?

Reply:
This should depend only on workspace context. From ws context org should be picked up. 

Only workspace context should have local storage connection